win/cubridsa/cubridsa.def

```diff
@@ -1,831 +1,831 @@
+    db_find_or_create_session
+    db_end_session
+    db_set_server_session_key
+    db_get_server_session_key
+    db_get_variable
+    db_set_get
+    db_set_size
+    db_set_page_size
+    db_shutdown
+    db_string_free
+    db_trigger_action
+    db_trigger_action_time
+    db_trigger_action_type
+    db_trigger_attribute
+    db_trigger_class
+    db_trigger_comment
+    db_trigger_condition
+    db_trigger_condition_time
+    db_trigger_event
+    db_trigger_name
+    db_trigger_priority
+    db_trigger_status
+    do_is_partitioned_subclass
+    er_errid
+    er_errid_if_has_error
+    fileio_make_volume_info_name
+    locator_has_heap
+    ntohi64
+    numeric_coerce_int_to_num
+    pgbuf_fix_release
+    pgbuf_fix_debug
+    pgbuf_get_vpid_ptr
+    pgbuf_set_dirty
+    prm_get_master_port_id
+    prm_get_bool_value
+    prm_get_float_value
+    prm_get_integer_list_value
+    prm_get_integer_value
+    prm_get_bigint_value
+    prm_get_string_value
+    prm_get_name
+    rel_copy_version_string
+    rel_build_number
+    rel_build_os
+    rel_disk_compatible
+    rel_name
+    rel_release_string
+    sm_mark_system_class
+    sm_finish_class
+    sm_update_class
+    smt_add_attribute
+    smt_add_class_method
+    smt_def_class
+    smt_edit_class_mop
+    smt_quit
+    smt_set_attribute_default
+    sysprm_load_and_init
+    sysprm_set_to_default
+    sysprm_set_force
+    sysprm_obtain_parameters
+    sysprm_set_er_log_file
+    sysprm_check_range
+    sysprm_get_range
+    timezone_compile_data
+    utility_get_generic_message
+    util_make_ha_conf
+    util_get_ha_mode_for_sa_utils
+    util_is_localhost
+    util_free_ha_conf
+    utility_initialize
+    util_byte_to_size_string
+    util_size_string_to_byte
+    util_log_write_result
+    util_log_write_errid
+    util_log_write_errstr
+    util_log_write_command
+    au_get_user_class_name
+    tr_get_class_name
+    boot_emergency_patch
+    boot_set_skip_check_ct_classes
+    ml_ext_free
+    ml_ext_add
+    ml_size
+    sm_get_ch_name
+    tr_map_trigger
+    er_init
+    er_final
+    jsp_connect_server
+    jsp_disconnect_server
+    jsp_writen
+    jsp_readn
+    jsp_start_server
+    jsp_server_port
+    javasp_get_info_file
+    javasp_get_error_file
+    javasp_open_info_dir
+    javasp_open_info
+    javasp_read_info
+    javasp_reset_info
+    javasp_write_info
+;
+; utility functions
+;
+	createdb
+	renamedb
+	copydb
+	deletedb
+	backupdb
+	restoredb
+	addvoldb
+	spacedb
+	optimizedb
+	installdb
+	diagdb
+	patchdb
+	checkdb
+	alterdbhost
+	paramdump
+	changemode
+	copylogdb
+	applylogdb
+	statdump
+	acldb
+	genlocale
+	dumplocale
+	gen_tz
+	dump_tz
+	synccolldb
+	synccoll_force
+	restoreslave
+	vacuumdb
+
+;
+;	enable below two functions when
+;	 #if defined(ENABLE_UNUSED_FUNCTION) is on
+;
+;	estimatedb_data
+;	estimatedb_index
+;
+
+	loaddb_user
+	unloaddb
+	compactdb
+	csql
+	csql_get_message
+	basename
+	au_set_password
+	au_disable_passwords
+	er_has_error
```

Comment:
Is this kind of tap space changes are intended?

Reply:
yes, the indentation was broken.